openmcyclus/depletion.py

```diff
@@ -183,12 +167,14 @@ def get_spent_comps(self, material_ids, path):
         results = od.Results(path + "depletion_results.h5")
         spent_comps = []
         for index, material_id in enumerate(material_ids):
-            material = results[-1].get_material(material_id)
+            spent_comp = results[-1].get_material(str(material_id))
+
             comp = {}
-            for nuclide in material.nuclides:
-                if nuclide.percent < 1e-15:
+            for nuclide in spent_comp.nuclides:
+                if nuclide.percent < 1e-5:
```

Comment:
The threshold is somewhat arbitrary. It's to make sure you don't really track nuclides that are at effectively zero mass. I guess it could be user defined but I also wanted to limit the number of user definitions to make it easier to use.

Reply:
Okay that's reasonable